src/local.py

```diff
@@ -26,14 +26,21 @@ def __init__(self):
 
     def get_local_launcher_path(self):
         try:
-            # The uninstall key for the launcher is called Rockstar Games Launcher.
-            key = OpenKey(self.root_reg, WINDOWS_UNINSTALL_KEY + "Rockstar Games Launcher")
-            dir, type = QueryValueEx(key, "InstallLocation")
-            self.installer_location = dir[:-1] + "\\Launcher.exe\""
-            if LOG_SENSITIVE_DATA:
-                log.debug("ROCKSTAR_INSTALLER_PATH: " + self.installer_location)
+            if CONFIG_OPTIONS['rockstar_launcher_path_override']:
+                self.installer_location = CONFIG_OPTIONS['rockstar_launcher_path_override']
+                if LOG_SENSITIVE_DATA:
```

Comment:
Side note: I don't think local paths are sensitive data. Logging them may be helpful for fixing problems in the future